tasktiger/worker.py

```diff
@@ -344,11 +429,9 @@ def _execute(self, queue, tasks, log, locks, all_task_ids):
         else:
             # Main process
             log = log.bind(child_pid=child_pid)
-            log.debug('processing', func=task_func, params=[{
-                    'task_id': task.id,
-                    'args': task.args,
-                    'kwargs': task.kwargs,
-            } for task in tasks])
+            for task in tasks:
+                log.info('processing', func=task_func, task_id=task.id,
+                         params={'args': task.args, 'kwargs': task.kwargs})
```

Comment:
When relying on json log output you want to be able to filter by `task_id` and see when the task was started, done, and/or errored.  The `processing` event had multiple `task_id`'s in the `params` array which means you need to filter differently for the same task depending on if you are searching for dones and errors versus the processing event.

This isn't really an issue if you are just grepping for a specific task id in a log file but if you are using something like Kibana to do the search it means you can unknowingly miss log entries when you query `task_id=XYZ`.